submodules/LIB/hardware/syn/build.tcl

```diff
@@ -25,7 +25,7 @@ set_db stdout_log genus.log
 set_db information_level 7
 set_db super_thread_debug_directory st_part_log
 
-set_db hdl_error_on_latch true 
+set_db hdl_error_on_latch false 
```

Comment:
This should have an IF, we still want other hardware to give an error on latch

Reply:
ok will add it

Reply:
tried to use pragmas but didnt work, i dont have an ideia how to solve without creating a synthesisis script just for pico32.